<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BitCoin AUTO v0.33</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>511</width>
      <height>121</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>가상화폐</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>현재가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>시가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>목표가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>보유여부</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>알고리즘 매매 시작</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>알고리즘 매매 중지</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>9</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>14</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>기준 시간</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>511</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>14</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>매도 시간</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>9</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
    <property name="time">
     <time>
      <hour>23</hour>
      <minute>59</minute>
      <second>0</second>
     </time>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>532</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
